ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/CreateTableCommand.java

```diff
@@ -35,15 +35,14 @@
 
   @Override
   public DdlCommandResult run(final MutableMetaStore metaStore) {
-    if (registerTopicCommand != null) {
-      try {
-        registerTopicCommand.run(metaStore);
-      } catch (KsqlException e) {
-        final String errorMessage =
-                String.format("Cannot create table '%s': %s", topicName, e.getMessage());
-        throw new KsqlException(errorMessage, e);
-      }
+    try {
+      createTopic(metaStore);
+    } catch (KsqlException e) {
```

Comment:
As above, we can drop this try catch and replace with `registerTopic(metaStore, "table")`

Reply:
Done